### PR TITLE
tesseract-ocr: fix clang build

### PR DESCRIPTION
**Pull request by raedrizqie**

### CODE REVIEW

File: mingw-w64-tesseract-ocr/PKGBUILD [hunk 1]
```diff
@@ -22,55 +23,57 @@ depends=(${MINGW_PACKAGE_PREFIX}-cairo
...
-  ../tesseract-${pkgver}/configure \
+  export LIBLEPT_HEADERSDIR="${MINGW_PREFIX}/include"
+
+  ./configure \
     --build=${MINGW_CHOST} \
     --host=${MINGW_CHOST} \
     --target=${MINGW_CHOST} \
     --prefix=${MINGW_PREFIX} \
-    LIBLEPT_HEADERSDIR=${MINGW_PREFIX}/include \
+    --disable-openmp \
```
Comment (jeremyd2019) [inline mingw-w64-tesseract-ocr/PKGBUILD:59]: Maybe only disable openmp for clang, or else add explicit linking to `-lomp` there to work around #8554?

**Comment by lazka** [comment]
hm, it failed in autobuild:

```
/libtesseract_fma.la ../arch/libtesseract_sse.la 
2021-11-14T13:25:01.6966267Z libtool:   error: CURRENT '' must be a nonnegative integer
2021-11-14T13:25:01.7428522Z libtool:   error: ':' is not valid version information
2021-11-14T13:25:01.7447809Z make[2]: *** [Makefile:602: libtesseract.la] Error 1
2021-11-14T13:25:01.7449211Z make[2]: Leaving directory '/c/M/mingw-w64-tesseract-ocr/src/build-MINGW64/src/api'
2021-11-14T13:25:01.7513281Z make[1]: *** [Makefile:504: all-recursive] Error 1
2021-11-14T13:25:01.7514259Z make[1]: Leaving directory '/c/M/mingw-w64-tesseract-ocr/src/build-MINGW64'
2021-11-14T13:25:01.7517673Z make: *** [Makefile:413: all] Error 2
2021-11-14T13:25:01.8217218Z ==> ERROR: A failure occurred in build().
```

**Comment by raedrizqie** [comment]
i think you merged the wrong commit.. it should be `a76230d` not `db19e42`

**Comment by lazka** [comment]
I just clicked "merge" :)

**Comment by raedrizqie** [comment]
maybe it was because the force push

**Comment by lazka** [comment]
there is no difference between this branch and master, so must be something different

**Comment by raedrizqie** [comment]
yeah.. you are right.. but why it passes all the checks here, but failed when merged?

i will try to look at it

**Comment by raedrizqie** [comment]
new PR: #10091

**Pull request merged**